internal/versions/versions_test.go

```diff
@@ -70,12 +70,20 @@ var defaultOsImages = models.OsImages{
 }
 
 var defaultReleaseImages = models.ReleaseImages{
+	&models.ReleaseImage{
+		// This image uses a syntax with missing "cpu_architectures". It is crafted
+		// in order to make sure the change in MGMT-11494 is backwards-compatible.
+		CPUArchitecture:  swag.String("fake-stripped-architecture"),
```

Comment:
What does stripped mean in this context? The meaning isn't obvious to me, so if you can think of a different english word(s) to describe it it could be more clear

Reply:
Just removed it. With adding explicit empty array it should be clear enough